component.json

```diff
@@ -13,12 +16,70 @@
       }
     },
     "jsonToXml": {
-      "title": "JSON to XML",
-      "main": "./lib/actions/jsonToXml.js",
+      "title": "JSON to XML (Deprecated)",
```

Comment:
maybe it makes sense remove `(Deprecated)` from the title, as we already have `"deprecated": true`?